src/lib-components/CardMeta.vue

```diff
@@ -167,8 +167,18 @@ const classes = computed(() => {
     <h3
       v-else
       class="title-no-link"
-      v-html="title"
```

Comment:
what is the reason for this change? Was it a linter or intentional? This will break any uses of CardMeta where a html link was injected with v-html